examples/run/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 set(TARGET llama-run)
 add_executable(${TARGET} run.cpp)
 install(TARGETS ${TARGET} RUNTIME)
-target_link_libraries(${TARGET} PRIVATE llama ${CMAKE_THREAD_LIBS_INIT})
+target_link_libraries(${TARGET} PRIVATE common llama ${CMAKE_THREAD_LIBS_INIT})
+target_include_directories(${TARGET} PUBLIC ../../common)
```

Comment:
This include should not be necessary. The `common` target should carry the necessary include paths.

Reply:
Changed